_src/static/sass/components/_copy.scss

```diff
@@ -0,0 +1,118 @@
+// ------------------------------
+// edX Pattern Library: Components - Copy
+
+// About: Contains base styling for copy (general text and type used in content and functionaly).
+
+// #GLOBAL
+// #INDIVIDUAL COPY CASES
+
+// ----------------------------
+// #GLOBAL
+// ----------------------------
+.copy {
+    @extend %reset-copy;
```

Comment:
Ah, I see it is only font-family now which isn't a big deal. Either way, good sir!